File: Java/src/World/body/animal/Wolf.java
package World.body.animal;

import GUI.Color_obj;
import World.Point;
import World.World;
import World.body.Animal;
import World.body.Body;

import java.io.Serializable;

public class Wolf extends Animal implements Serializable {
    private static final long serialVersionUID = 42L;
    private static int amount=0;
    public Wolf(Point point_location, World world) {
        super(point_location, world);
        amount ++;
        setName("Wolf");
        setColor(Color_obj.WOLF);
        setPower(9);
        setInitiative(5);
    }
    public static int getAmount_wolf() {
        return amount;
    }

    @Override
    protected boolean repel_attack(Body attacker) {
        return false;
    }
}

File: Python/World/Body/Animal/wolf.py
from GUI.colors import Colors
from World.Body.animal import Animal


class Wolf(Animal):

    def __init__(self, point_location, world):
        super().__init__(point_location, world)
        self.set_name("Wolf")
        self.set_color(Colors.WOLF)
        self.set_power(9)
        self.set_initiative(5)

    def repel_attack(self, attacker):
        return False

